<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>随心播</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>1241</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tiLiveList">
     <attribute name="title">
      <string>直播列表</string>
     </attribute>
     <widget class="QPushButton" name="btnRefreshLiveList">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>10</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新直播列表</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLastPage">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>60</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>上一页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNextPage">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>110</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>下一页</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbPageNum">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>170</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0/0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1121</width>
        <height>561</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="layoutShowRoomList"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tiBeginLive">
     <attribute name="title">
      <string>我要直播</string>
     </attribute>
     <widget class="QLabel" name="lbBLTitle">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>标题:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="edBLTitle">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PC随心播</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBeginLive">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>711</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>开始直播</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnLogin">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edPassword">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbPassword">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>密码:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edUserName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbUserName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRegister">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edUserName</tabstop>
  <tabstop>edPassword</tabstop>
  <tabstop>btnLogin</tabstop>
  <tabstop>btnRegister</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnRefreshLiveList</tabstop>
  <tabstop>btnLastPage</tabstop>
  <tabstop>btnNextPage</tabstop>
  <tabstop>edBLTitle</tabstop>
  <tabstop>btnBeginLive</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
